v2/cf-nodejs-logging-support/src/lib/config/config.ts

```diff
@@ -0,0 +1,143 @@
+import { MergedConfig, ConfigObject, ConfigField, customFieldsFormat } from './interfaces';
+import * as coreConfig from './config-core.json';
+import * as requestConfig from './config-request.json';
+import * as cfConfig from './config-cf.json';
+import * as kymaConfig from './config-kyma.json';
+import * as appLoggingConfig from './config-app-logging.json';
+import * as cloudLoggingConfig from './config-cloud-logging.json';
+import EnvService from '../core/env-service';
+
+export default class Config {
+
+    private static instance: Config;
+
+    private config: MergedConfig = {
+        "fields": [],
+        "customFieldsFormat": "cloud-logging",
+        "outputStartupMsg": false
+    }
+
+    private constructor() {}
+
+    public static getInstance(): Config {
+        if (!Config.instance) {
+            let env = EnvService.getRuntimeName();
+            let boundServices = EnvService.getBoundServices();
+            let envConfig = () => {
+                if (env == "Kyma") {
```

Comment:
Is this working like this?
on the same note, maybe this variable could be reflected colectively in a configuration instead of magic constants in code.

Reply:
changed this to a list of config objects (as suggested in the other conversation) that is builded sequentially to make it more readable.